client/main/results.js

```diff
@@ -30,18 +30,20 @@ var ResultsView = Backbone.View.extend({
             model: this.model,
             mode: 'text' });
 
-        this.$welcome = $('<iframe id="main_welcome" \
-                name="welcome" \
-                sandbox="allow-scripts allow-same-origin" \
-                src="https://jamovi.org/welcome.html?version=0.7.3.0" \
-                class="silky-welcome-panel" \
-                style="overflow: hidden; box-sizing: border-box;" \
-                ></iframe>');
-        this.$welcome.appendTo(this.$el);
-
-        this.model.analyses().once('analysisResultsChanged', (event) => {
-            this.$welcome.addClass('silky-welcome-panel-hidden');
-        });
+        if (this.model.analyses().analysesLoaded() === false) {
```

Comment:
when the control is initialized, none of the analyses will have loaded.

to fix this, i'd delay this code by 500ms